<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="group_box_file_system_tree">
       <property name="title">
        <string>Выберите папку</string>
       </property>
       <layout class="QHBoxLayout" name="horizontal_layout">
        <item>
         <widget class="QTreeView" name="tree_view_file_system"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontal_layout_2">
          <item>
           <widget class="QLabel" name="label_folder_path">
            <property name="text">
             <string>Выбранная папка</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_edit_folder_path"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="table_statistics"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
